app/Models/Bible/Bible.php

```diff
@@ -367,6 +368,29 @@ function ($join) use ($formatted_search) {
             );
     }
 
+     /**
+     * Add join related to the bible_translations and searching by ID
+     *
+     * @param Builder $query
+     * @param string $bible_id_query
+     *
+     * @return Builder
+     */
+    public function scopeMatchByBibleId(Builder $query, string $bible_id_query)
+    {
+        $formatted_search = "%$bible_id_query";
```

Comment:
Note that putting the wildcard at the beginning will defeat any index we create.
I'll allow it in this case since there are only 3000+ rows in the table scan

Reply:
@bradflood , I understand. I have used the wildcard because it was faster than use RIGHT or LOCATE operators:

```sql
and RIGHT(ver_title.bible_id,3) = 'kjv'
and LOCATE('kjv', ver_title.bible_id) > 0
```